parity/configuration.rs

```diff
@@ -186,6 +187,7 @@ impl Configuration {
 				file_path: self.args.arg_file.clone(),
 				wal: wal,
 				kind: snapshot::Kind::Restore,
+				block_at: try!(to_block_id("latest")), // unimportant.
```

Comment:
i can refactor the common fields out into a struct of its own, but then you end up with ~5 extra lines of boilerplate as opposed to one.